### ISSUE TITLE
[Question] 

### ISSUE THREAD

**Issue by DarthAhsoka** [finallyfunctional/openvr-driver-example#1]
Excuse me
       Could I ask for advicea some question for this sample? I make a new hardware Controller, It's built-in gyroscope and Wear it on your body， I want  this controller just replace left hand joystick，the left hand joystick other function need stay(like home button) , the new controller upload the data
from BLE signal,  I reciever this data and convert to move-forward, move-back,move-left or move-rgiht , I need write this code in RunFrame() ? 
       thank you very mush .

**Comment by finallyfunctional** [comment]
Yup. Get your input from BLE and use it to update the joystick X and Y scalar components in the ControllerDriver.RunFrame() method.

**Comment by DarthAhsoka** [comment]
I study develop openvr  driver from your sample and video,  thank you very much, 

 I have a few question in my ENV,  i use windows mixed reality and Samsung MR glass, In VR app, I set controller config , use custom setting for my driver, but it's not take effective, the controller settings auto set default always, how to deal with it ? thank you
